test/test_objects_with_sets.py

```diff
@@ -0,0 +1,36 @@
+"""Tests of the ObjectsWithSets class."""
+
+from typing import Any
+
+import pytest
+from installed_clients.WorkspaceClient import Workspace
+from StaticNarrative.exporter.dynamic_service_client import DynamicServiceClient
+from StaticNarrative.exporter.objects_with_sets import ObjectsWithSets
+
+
+@pytest.fixture()
+def set_api_client(config: dict[str, Any], token: str) -> DynamicServiceClient:
+    """Get a client for the SetAPI."""
+    return DynamicServiceClient(config["srv-wiz-url"], "release", "SetAPI", token)
+
+
+@pytest.fixture()
+def ows(
+    workspace_client: Workspace, set_api_client: DynamicServiceClient, token: str
+) -> ObjectsWithSets:
+    """Get an instance of ObjectsWithSets."""
+    return ObjectsWithSets(
+        set_api_client=set_api_client, workspace_client=workspace_client, token=token
+    )
+
+
+def test_set_api_client_local_vs_set_api_remote(ows: ObjectsWithSets) -> None:
+    """This is not really a genuine test -- it just ensures that the local vs remote calls return the same result."""
```

Comment:
this is a crappy test - I just put it in place to be able to check that the local vs the remote SetAPI things return the same result.

I have some refinements to make on ObjectsWithSets / the GenericSetNavigator in upcoming PRs, so I'll probably make a few more of these crappy tests just to check that the output remains the same even if the innards of the method change.